lib/safira/email.ex

```diff
@@ -12,9 +12,10 @@ defmodule Safira.Email do
   end
 
   def send_registration_email(to_email, token, discord_association_code) do
+    Mix.shell().info to_email
     new_email()
```

Comment:
Figures that if we're sending 600+ emails we should be a bit more aware of what's going on